value.go

```diff
@@ -1000,14 +1000,26 @@ func (vlog *valueLog) createVlogFile(fid uint32) (*logFile, error) {
 		return nil, errFile(err, lf.path, "Create value log file")
 	}
 
+	removeFile := func() {
```

Comment:
Add test for this.

Reply:
I tried to add a test by setting ulimit but this will be difficult for two reasons
1. `ulimit` is set per process. If I set the `ulimit` for a test, it might affect other tests too.
2. It's hard to set the correct `ulimit` and the test wouldn't fail even if the limit is off by 1.